include/z64scene.h

```diff
@@ -433,7 +433,7 @@ typedef struct {
 
 typedef struct {
     /* 0x00 */ u16 unknown;
-    /* 0x02 */ s8 bgID;
+    /* 0x02 */ s8 bgId;
```

Comment:
Every cleanup you've done in `z64scene.h` are on structs that are deleted in https://github.com/zeldaret/mm/pull/1143, so might as well revert them just to prevent conflicts